aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -4,12 +4,12 @@ author: camsoper
 description: Discover how to set up Azure Active Directory B2C authentication with ASP.NET Core.
 ms.author: casoper
 manager: wpickett
-ms.date: 01/12/2018
+ms.date: 01/25/2018
 ms.topic: tutorial
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: security/authentication/azure-ad-b2c
-custom: mvc
+ms.custom: mvc
```

Comment:
Move this up (alphabetical in the middle or just someplace in the middle) ... Whatever we decide to do here with my script later, I hope we keep the title first and the UID last.